docusearch/templates/search/search.html

```diff
@@ -24,13 +24,31 @@
 
         <div>
         <dl>
-        <h3><a href="?q={{request.GET.q}}&amp;order_by=date_released">Order By</a> </h3>
+        <dd>
+            {% if request.GET.order_by == 'date_released' %}
+            <a href="{{generate_get_req([
```

Comment:
This is a lot of logic in the templates. Should we move the generate of these links to the view? That way the code's testable.